iOS/ViewControllers/Account/AppearanceSettingsViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AppearanceSettingsViewController: UITableViewController {
+
+    @available(iOS 13.0, *)
+    private var theme: Theme {
+        get {
+            return UserDefaults.standard.theme
+        }
+        set {
+            UserDefaults.standard.theme = newValue
+            self.configureStyle(for: newValue)
+        }
+    }
```

Comment:
You mainly use this attribute for checks like `indexPath.row == self.theme.rawValue` to avoid setting the same appearance again. IMO you can remove this optimization and, thus, this attribute as the user will not change the appearance setting on a daily basis and the appearance change is not a resource-intensive operation.